### PR TITLE
OSBS - commit repos directory even if its removed

### LINKED ISSUE #103: not committing removal of stale .repo files
Using concreate I just tried a build of [jboss-openjdk-image](https://github.com/jboss-container-images/jboss-openjdk-image), the resulting changes removed some .repo files, e.g.

```
 # Install required RPMs and ensure that the packages were installed
-RUN yum install -y --disablerepo=\* --enablerepo=jboss-rhel-optional --enablerepo=jboss-rhel-os --enablerepo=jboss-rhel-ose --enablerepo=jboss-rhel-rhscl java-1.8.0-openjdk-1.8.0.141 java-1.8.0-openjdk-headless-1.8.0.141 java-1.8.0-openjdk-devel-1.8.0.141 \
+RUN yum install -y --disablerepo=\* --enablerepo=jboss-rhel-os java-1.8.0-openjdk-1.8.0.141 java-1.8.0-openjdk-headless-1.8.0.141 java-1.8.0-openjdk-devel-1.8.0.141 \
     && yum clean all && \
     rpm -q  java-1.8.0-openjdk-1.8.0.141 java-1.8.0-openjdk-headless-1.8.0.141 java-1.8.0-openjdk-devel-1.8.0.141
 # Remove custom repo files
-RUN rm /etc/yum.repos.d/jboss-rhel-optional.repo /etc/yum.repos.d/jboss-rhel-os.repo /etc/yum.repos.d/jboss-rhel-ose.repo /etc/yum.repos.d/jboss-rhel-rhscl.repo
-
+RUN rm /etc/yum.repos.d/jboss-rhel-os.repo
```

The build failed afterwards because the repo files removal was staged for commit but not committed

```
2017-10-13 12:38:39,637 concreate    DEBUG    Executing 'rhpkg container-build'.
Could not execute container_build_koji: /home/ce/.concreate.d/osbs/jboss-openjdk-docker has uncommitted changes.  Use git status to see details
Traceback (most recent call last):
  File "/usr/bin/concreate", line 9, in <module>
    load_entry_point('concreate==1.1.7', 'console_scripts', 'concreate')()
  File "/usr/lib/python2.7/site-packages/concreate/cli.py", line 158, in run
    Concreate().run()
  File "/usr/lib/python2.7/site-packages/concreate/cli.py", line 134, in run
    builder.build(self.args)
  File "/usr/lib/python2.7/site-packages/concreate/builders/osbs.py", line 101, in build
    subprocess.check_call(build_cmd)
  File "/usr/lib64/python2.7/subprocess.py", line 542, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['rhpkg', 'container-build']' returned non-zero exit status 1
```

Looking at the back-end dist git repo, the three repo files were deleted but not added into the sync commit (I've manually redacted the diff hunks from below because they have private URLs)

```
  cevm  ce   jb-openjdk18-rhel7-1.1  ~  .concreate.d  osbs  jboss-openjdk-docker  git diff
diff --git a/repos/jboss-rhel-optional.repo b/repos/jboss-rhel-optional.repo
deleted file mode 100644
index 08e3a87..0000000
--- a/repos/jboss-rhel-optional.repo
+++ /dev/null

diff --git a/repos/jboss-rhel-ose.repo b/repos/jboss-rhel-ose.repo
deleted file mode 100644
index 0f3ba19..0000000
--- a/repos/jboss-rhel-ose.repo
+++ /dev/null

diff --git a/repos/jboss-rhel-rhscl.repo b/repos/jboss-rhel-rhscl.repo
deleted file mode 100644
index 9cdede2..0000000
--- a/repos/jboss-rhel-rhscl.repo
+++ /dev/null
```

### PR DESCRIPTION
**Pull request by dbecvarik**
fixes #103

### CODE REVIEW

**Comment by dbecvarik** [comment]
also should fix #84

File: concreate/builders/osbs.py [hunk 1]
```diff
@@ -171,9 +171,9 @@ def add(self):
         # Add new Dockerfile
         subprocess.check_call(["git", "add", "Dockerfile"])
 
-        for d in ["repos", "modules"]:
-            if os.path.exists(os.path.join(self.output, d)):
-                subprocess.check_call(["git", "add", d])
+        for d in ["repos", "modules", "ASAS"]:
```
Comment (goldmann) [inline concreate/builders/osbs.py]: "ASAS"?

File: concreate/builders/osbs.py [hunk 2]
```diff
@@ -206,7 +206,8 @@ def commit(self):
             subprocess.call(["git", "show"])
 
         if not (self.noninteractive or decision("Are you ok with the changes?")):
-            subprocess.call(["bash"])
+            ps1 = os.getenv("PS1", "$ ")
+            subprocess.call(["bash"], env={'PS1': "concreate-%s" % ps1})
```
Comment (goldmann) [inline concreate/builders/osbs.py]: This is a nice idea!

Additionally we should think about writing some "into" text.

File: concreate/builders/osbs.py (hunk 1 above)
Comment (dbecvarik) [reply concreate/builders/osbs.py]: I've testes something aleft it here, fixing

**Pull request merged**